__tests__/app.test.js

```diff
@@ -69,3 +69,34 @@ describe("/api", () => {
     });
   });
 });
+
+describe("PATCH /api/articles/:article_id ", () => {
+  test("201: returns an object with updated votes property by the provided number", () => {
+    const req = { inc_votes: 3 };
+    const expected = {
+      article_id: 1,
+      title: "Living in the shadow of a great man",
+      topic: "mitch",
+      author: "butter_bridge",
+      body: "I find this existence challenging",
+      created_at: expect.any(Date),
+      votes: 103,
+    };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(req)
+      .expect(201)
+      .then(({ body }) => {
+        const formattedResponse = convertTimestampToDate(body.updatedVotes);
+        expect(formattedResponse).toEqual(expected);
+      });
+  });
```

Comment:
Can we please have another test for an article id that is of the wrong type? E.g /api/articles/notAnId

Reply:
Good point. Thank you. Give me a minute. :)